core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -245,7 +245,17 @@ object TopicCommand extends Logging {
 
         newTopic.configs(configsMap)
         val createResult = adminClient.createTopics(Collections.singleton(newTopic))
-        createResult.all().get()
+        try {
+          createResult.all().get()
```

Comment:
nit: the parenthesis are not necessary. there is other cases when they can be removed.